protected/tests/unit/controllers/SiteTest.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+class SiteTest extends CDbTestCase
+{
+	public function testSearchBeers()
+	{
```

Comment:
Stubs left to show the test coverage I believe would be required to make this a complete merge request, sadly was unable to get unit tests working.